### ISSUE TITLE
change Background Color

### ISSUE THREAD

**Issue by diniremix** [biigpongsatorn/vue-element-loading#8]
Hello,
excellent app :+1: , I have a question, is it possible to change the color of the **background**, by means of a property in **vue-element-loading**, as well as a color is passed in the color property?

Example:

```html
<vue-element-loading :active="isActive" spinner="bar-fade-scale" color="#FF6700" background="#1a1b1c"/>
```

in the file: **src/index.vue**
 
```css
.velmld-overlay {
  background-color: rgba(255, 255, 255, .9);
}
```

Thanks!

**Comment by biigpongsatorn** [comment]
Yes it a good idea, Can you make a pr for this feature ?

**Comment by runyasak** [comment]
I have create a pull request that added background color.
Please, checkout on this PR #10

**Comment by biigpongsatorn** [comment]
@runyasak  Thanks

**Comment by biigpongsatorn** [comment]
@diniremix  Published to `1.0.6`
